com.ca.lsp.cobol/lsp-core-domain/src/main/java/com/broadcom/lsp/domain/cobol/event/api/CopybookObserver.java

```diff
@@ -14,12 +14,12 @@
  *
  */
 
-package com.broadcom.lsp.domain.cobol.databus.api;
+package com.broadcom.lsp.domain.cobol.event.api;
 
 /**
  * This interface implements the Observer pattern for the databus. It provides a callback method to
  * perform some action if the required event is populated.
  */
-public interface DataBusObserver<O> {
+public interface CopybookObserver<O> {
```

Comment:
I don't think that CopybookObserver is a good name because the purpose of this mechanism is to allow communications between different modules and not only for the copybooks. 
For example, I am going to use it to allow calls from the WorkspaceService to the TextDocumentService after triggering by the watcher.